### PR TITLE
enhancement: change the date format in widget 

### LINKED ISSUE #800: Enhancement: Show the date in today, yesterday or other date format in widget

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes #800 

**Preview**

Screenshot 2020-12-18 at 7 43 12 PM

### CODE REVIEW

File: app/javascript/shared/helpers/DateHelper.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 import format from 'date-fns/format';
+import isToday from 'date-fns/isToday';
+import isYesterday from 'date-fns/isYesterday';
 
 export const formatUnixDate = (date, dateFormat = 'MMM dd, yyyy') => {
   const unixDate = fromUnixTime(date);
   return format(unixDate, dateFormat);
 };
+
+export const formatDate = ({ date, todayText, yesterdayText }) => {
```
Comment (pranavrajs) [inline app/javascript/shared/helpers/DateHelper.js:11]: Please add specs for this function.

File: app/javascript/shared/components/DateSeparator.vue [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+import { formatDate } from 'shared/helpers/DateHelper';
 export default {
   props: {
     date: {
       type: String,
       required: true,
     },
   },
+  methods: {
+    formatedDate() {
```
Comment (pranavrajs) [inline app/javascript/shared/components/DateSeparator.vue:17]: Please use computed properties instead of a method.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Nice work! I have some comments, please check the individual comments.

**Comment by muhsin-k** [comment]
@pranavrajs Please check the latest changes

File: app/javascript/shared/components/DateSeparator.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="date--separator">
-    {{ formatedDate() }}
+    {{ formatedDate }}
```
Comment (pranavrajs) [inline app/javascript/shared/components/DateSeparator.vue:3]: Spelling mistake.

**Pull request merged**